ixmp/core.py

```diff
@@ -1137,6 +1137,10 @@ def clone(self, model=None, scenario=None, annotation=None,
         first_model_year: int, optional
             new first model year in cloned scenario
             ('slicing', only available for MESSAGE-scheme scenarios)
```

Comment:
Sorry, the request was to *remove* the two original lines describing this argument: because they mention 'MESSAGE-scheme' where they should actually not.